src/garlic-toast.md

```diff
@@ -0,0 +1,27 @@
+# Garlic Toast
+
+Garlic toast, perfect as a substitute for crackers in a cheese platter.
+
+- ⏲️ Prep time: 5 min
+- 🍳Cook time: 10-15 min
+
+## Ingredients
+
+- Sourdough loaf (sliced)
+- Garlic-infused olive oil
+- Garlic salt
+
+## Directions
+
+1. Preheat oven to 180°C (356°F).
+2. Place sliced sourdough on a baking tray, prepared with aluminium foil

```

Comment:
```suggestion
2. Place sliced bread on a baking tray, covered with aluminum foil

```

Reply:
`aluminium` is actually the correct spelling for most of the world. I think Canada and the US may be the only places that don't. So I guess the question is, do we want to force American spelling.